tests/test_utils_persistence.py

```diff
@@ -33,6 +33,32 @@ def test_swallows_exceptions_making_new_directories(mocker):
     os.makedirs.assert_called_once()
 
 
+def test_setifying_list_returns_expected_set():
+    assert persistence.setify(['ABC', 2]) == {'ABC', 2}
+
+def test_setifying_string_returns_expected_single_item_set():
+    assert persistence.setify('ABC') == {'ABC'}
+
+def test_setifying_set_makes_no_difference():
+    assert persistence.setify({'ABC', 2}) == {'ABC', 2}
```

Comment:
Do we expect the original set, or a semantically equivalent copy of that set?

Reply:
yeah it won't be the original set, but I only care about the contents 😄